app/rest/dao/src/main/resources/io/syndesis/dao/deployment.json

```diff
@@ -1,179 +1,4 @@
 [
-  {
```

Comment:
I see, as only twitter has been ported, we still need deployment.json (wrt to my other comment)